src/cmd/run.go

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2023-Present The UDS Authors
+
+package cmd
+
+import (
+	"github.com/defenseunicorns/zarf/src/pkg/message"
+	"github.com/defenseunicorns/zarf/src/pkg/utils"
+	"github.com/spf13/cobra"
+
+	"github.com/defenseunicorns/uds-cli/src/config"
+	"github.com/defenseunicorns/uds-cli/src/config/lang"
+	"github.com/defenseunicorns/uds-cli/src/pkg/runner"
+	"github.com/defenseunicorns/uds-cli/src/types"
+)
+
+// runCmd represents the run command
+var runCmd = &cobra.Command{
+	Use:   "run [ TASK NAME ]",
+	Short: "run a task",
+	Long:  `run a task from an tasks file`,
+	Args:  cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		var tasksFile types.TasksFile
+
+		err := utils.ReadYaml(config.TaskFileLocation, &tasksFile)
+		if err != nil {
+			message.Fatalf(err, "%s not found", config.TaskFileLocation)
+		}
+
+		taskName := args[0]
+
+		if err := runner.Run(tasksFile, taskName); err != nil {
+			message.Fatalf(err, "Failed to run action: %s", err)
+		}
+	},
+}
+
+func init() {
+	rootCmd.AddCommand(runCmd)
+	runFlags := runCmd.Flags()
+	runFlags.StringVar(&config.TaskFileLocation, "file", config.TasksYAML, lang.CmdRunFlag)
```

Comment:
would love to see shorthand too.

```suggestion
	runFlags.StringVarP(&config.TaskFileLocation, "file", "f", config.TasksYAML, lang.CmdRunFlag)
```